.github/workflows/pyright.yml

```diff
@@ -29,9 +29,10 @@ jobs:
             source /home/runner/work/app/bin/activate
             pip install --upgrade setuptools==70.0.0
             pip install --upgrade pip==23.2.1
-            pip install -e 'git+https://github.com/open-data/ckan@canada-v2.10#egg=ckan' -r 'https://raw.githubusercontent.com/open-data/ckan/canada-v2.10/requirements.txt' -r 'https://raw.githubusercontent.com/open-data/ckan/canada-v2.10/dev-requirements.txt'
+            pip install -e 'git+https://github.com/open-data/ckan@canada-v2.10#egg=ckan' -r 'https://raw.githubusercontent.com/open-data/ckan/canada-v2.10/requirement-setuptools.txt' -r 'https://raw.githubusercontent.com/open-data/ckan/canada-v2.10/requirements.txt' -r 'https://raw.githubusercontent.com/open-data/ckan/canada-v2.10/dev-requirements.txt'
```

Comment:
if requirements change on a branch they won't be picked up as part of the pyright run until they are merged. Can we reference the files in `/home/runner/work/app/src/ckanext-canada` instead?

Reply:
that is for our `ckan` branch, the install of the `ckanext-canada` is done a couple lines below and uses the checkout action from the previous step.